pytorch_lightning/callbacks/model_checkpoint.py

```diff
@@ -578,20 +579,26 @@ def __resolve_ckpt_dir(self, trainer: "pl.Trainer") -> None:
         """
         if self.dirpath is not None:
             return  # short circuit
-
-        if trainer.logger is not None:
+        if trainer.loggers:
```

Comment:
I think we have the chance to make a decision here. Where do we want to store the checkpoints if there are multiple loggers, and no dirpath is passed to ModelCheckpoint, and no weights_save_path is passed to Trainer?

The options I see are:
1. (Current behavior) save checkpoints to default_root_dir
2. Save checkpoints to first logger in list
3. Save checkpoints to all loggers in list